libs/ledger-live-common/src/exchange/swap/completeExchange.ts

```diff
@@ -242,6 +261,20 @@ const completeExchange = (
   });
 };
 
+function isSPLTokenAccount(account: AccountLike): boolean {
+  return account.type !== "TokenAccount" && account.currency.id === "solana";
+}
+
+async function sendPKI(transport: Transport) {
+  // FIXME: version number hardcoded
+  const { descriptor, signature } = await calService.getCertificate(
+    transport.deviceModel!.id,
+    "1.3.0",
+  );
+
+  await loadPKI(transport, "TRUSTED_NAME", descriptor, signature);
+}
```

Comment:
Do we want to fix the missing model id and version number before merging this ?

Reply:
For your need, I can comment this function to quickly merge.